File: Codewars/8_kyu/Reversed_strings.js
// DESCRIPTION:
// Complete the solution so that it reverses the string passed into it.

// 'world'  =>  'dlrow'
// 'word'   =>  'drow'

// My Solution

function solution(str) {
    let arr = str.split('')
    arr = arr.reverse()
    let res = arr.join('')
    return res
}

File: Codewars/Go/8_kyu/Reversed Strings.go
// Complete the solution so that it reverses the string passed into it.

// 'world'  =>  'dlrow'
// 'word'   =>  'drow'

// My Solution:
package kata

func Solution(word string) (result string) {
  for _, v := range word{
    result = string(v) + result
  }
  return result
}